### PR TITLE
Make pacbio run name non editable to prevent clashes

### PR DESCRIPTION
**Pull request by stevieing**

### CODE REVIEW

File: src/components/PacbioRunInfo.vue [hunk 1]
```diff
@@ -4,11 +4,11 @@
     <b-row>
       <b-col>
         <b-form-input :value="runName"
-                @change="setName"
                 id="run_name"
                 placeholder="Run name"
                 type="text"
-                width="48"/>
+                width="48"
+                readonly />
```
Comment (harrietc52) [inline src/components/PacbioRunInfo.vue]: Nice !

File: tests/unit/store/traction/pacbio/runs/mutations.spec.js [hunk 2]
```diff
@@ -26,10 +26,12 @@ describe('mutateRun', () => {
   })
 
   it('can update the mutate the state, e.g setName', () => {
```
Comment (harrietc52) [inline tests/unit/store/traction/pacbio/runs/mutations.spec.js]: e.g. setSystemName

**Comment by stevieing** [comment]
> All good! The readonly name field on run creation, and then the auto generated name on edit is really nice
> 
> Just a few places to remove run from I think
> 
> PacbioRun.js:
> build()
> createRunPayload()
> updateRunPayload()
> 
> PacbioRunInfo:
> mapMutations setName
> 
> Wasn't sure if in the service we should remove permitting name from the runs controller too?

I wonder if we need to improve the test coverage to ensure that I don't miss those sort of things in the future.

We probably should remove permitting the name from the controller. I will add that as a new issue.

**Comment by harrietc52** [comment]
@stevieing Did the above get removed before merging, build, createRunPayload, updateRunPayload, setName etc?

**Pull request merged**